Themes/default/languages/Login.english.php

```diff
@@ -91,6 +95,7 @@
 $txt['setting_coppaPost_desc'] = 'Only applies if age restriction is in place';
 $txt['setting_coppaFax'] = 'Fax number to which approval forms should be faxed';
 $txt['setting_coppaPhone'] = 'Contact number for parents to contact with age restriction queries';
+$txt['setting_announcements_default'] = 'Enable "' . $txt['notify_announcements'] . '" by default.';
```

Comment:
We don't need this line in 2.1. The relevant setting already has a UI elsewhere.